dockerfiles/init/manifests/codenvy.env

```diff
@@ -86,17 +86,121 @@
 #CODENVY_SERVER_XMX=2048
 
 
-
 ########################################################################################
 #####                                                                              #####
-#####                                   DOCKER                                     #####
+#####                                 NETWORKING                                   #####
 #####                                                                              #####
-# IP Address
-#     The IP address of the Docker daemon that is running on your host. We do a 
-#     self-discovery to set this. You can combine this with values of DOCKER_HOST to 
-#     change communications from socket to TCP.
+#
+# Affects how the Codenvy server connects to the Docker Daemon, how the Che server and
+# workspaces establish connections to one another, and also how remote browser clients
+# should discover and connect to workspaces.
+#
+# Codenvy goes through a progression algorithm to establish the protocol, IP address and
+# port to establish communications with it is booting or starting a workspace.
+#
+# Browser --> Codenvy Server
+#    1. Default is 'http://localhost:${SERVER_PORT}/wsmaster/api'.
+#    2. Else use the value of CODENVY_API
```

Comment:
I would prefer to put rule that some value is taken from **var**, and default value in case that var is not set next **val**.